charts/shoot-core/charts/monitoring/charts/blackbox-exporter/templates/config.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: blackbox-exporter-config
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+    app: prometheus
+    role: monitoring
+data:
+  blackbox.yaml: |
+    modules:
+      http_apiserver:
```

Comment:
I don't see any authentication / bearer token. How is this working?

Reply:
Added tls config to the blackbox exporter.